vision/automl/edge_container_predict/automl_vision_edge_container_predict.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""This is an example to call REST API from TFServing docker containers.
+
+Examples:
+  python automl_vision_edge_container_predict.py \
+  --image_file_path=./test.jpg --image_key=1 --port_number=8051
+
+"""
+
+import argparse
+
+# [START automl_vision_edge_container_predict]
```

Comment:
will this code sample be used on the doc pages?

Reply:
yes. After this code is merged, we will update the current tutorial to add a link to this sample.